docs/sphinx/sections/api/apidocs/libraries/dagster-pipes.rst

```diff
@@ -1,51 +1,94 @@
 Pipes (dagster-pipes)
------------------------
-
-This library is intended for inclusion in an external process that integrates
-with Dagster using the Pipes protocol.
+=====================
 
 .. currentmodule:: dagster_pipes
 
+The ``dagster-pipes`` library is intended for inclusion in an external process that integrates with Dagster using the `Pipes </concepts/dagster-pipes>`_ protocol. This could be in an environment like Databricks, Kubernetes, or Docker. Using this library, you can write code in the external process that streams metadata back to Dagster.
+
+For a detailed look at the Pipes process, including how to customize it, refer to the `Dagster Pipes details and customization guide </concepts/dagster-pipes/dagster-pipes-details-and-customization#overview-and-terms>`_.
+
+**Looking to set up a Pipes client in Dagster?** Refer to the `Dagster Pipes API reference </_apidocs/pipes>`_.
+
+**Note**: This library isn't included with ``dagster`` and must be `installed separately <https://pypi.org/project/dagster-pipes/>`_.
+
+----
+
+Context
+-------
+
 .. autofunction:: open_dagster_pipes
 
 .. autoclass:: PipesContext
 
-.. autoclass:: DagsterPipesError
+----
```

Comment:
Can we indicate that everything below here is more advanced that uses using out-of-the-box integrations will not need to know about these really? All of this stuff is for protocol builders, which is a small subset of people.

Reply:
Addressed - lmk if you have specific wording in mind and I'll update it on both pages.